### ISSUE TITLE
Reproduction Question 

### ISSUE THREAD

**Issue by IandRover** [XavierJiezou/DiffCR#2]
Dear Authors,

Thank you for sharing your excellent paper and the complete codebase. We have a question regarding the reproduction and would greatly appreciate your guidance.

After following the instructions by running `python run.py -p test -c config/ours_sigmoid.json` and fixing several bugs, we observed an MAE of between 0.8674 and 0.9696 on the testing set, which is significantly higher than expected and reported. Could you kindly confirm if the code used for evaluation is accurate and whether the checkpoints are up-to-date?

Below are the changes we made to ensure execution.
1. Comment on all the code that causes syntax errors.
2. Change the data_root in `configs/ours_sigmoid.json` from pmaa to PMAA (where our data is)
3. Change and correct the package name in `/models/network_x0_dpm_solver`.
4. Add `alpha_t = alpha_t[:,None, None,None]` and  `sigma_t = sigma_t[:,None, None,None]` in `core/dpm_solver_pytorch.py` Line 292.

We look forward to your feedback and thank you in advance for your assistance.

Best regards,

**Comment by zhentao-zou** [comment]
Hello, I also reproduce the code in the dataset Sen2_MTC_New using 4 RTX3090,
The training loss is about 0.06 (mae metric) 
I have some question, I found the pixels of out images is negative, how to convert the mae metric to psnr, ssim metric
Or, you can chat me, email: zhtzou@gmail.com

**Comment by Ly403** [comment]
> Hello, I also reproduce the code in the dataset Sen2_MTC_New using 4 RTX3090, The training loss is about 0.06 (mae metric) I have some question, I found the pixels of out images is negative, how to convert the mae metric to psnr, ssim metric Or, you can chat me, email: [zhtzou@gmail.com](mailto:zhtzou@gmail.com)

Hello. I've also reproduced this code by modifying the code based on [@landRover](https://github.com/XavierJiezou/DiffCR/issues/2#issue-2286772914)'s techniques. Specifically, I used the weight *diffcr_new.pth* from the folder *pretrained*. Here are the results I obtained:

| mae | psnr | ssim | rmse|
|----|-----|----|----|
| 0.04189379885792732|26.9799747467041	|0.8941709995269775	|0.05102594569325447|

Note that the author didn't provide the implementation of PSNR, SSIM, and RMSE, so I implemented it on my own. The results do not match those reported in the paper, which is very weird. I'm unsure if the discrepancy is due to my error.

I have not trained this model yet. So, I'm curious, have you trained this model? Also, have you got the test results using your training weights?

I've also sent a letter to your email. You can also drop a letter to <a href="mailto:liuyi2052697@foxmail.com">liuyi2052697@foxmail.com</a>.

**Comment by Anorax24** [comment]
I tried reproducing the code but i am unable to import from the ours_double_encoder_splitcaCond_splitcaUnet file i have this file in the json format but i can only import modules if it is in the .py format what should i di

**Comment by zhr760** [comment]
Gentlemen, how can this mistake be corrected？
![image](https://github.com/user-attachments/assets/d52190c0-704f-4dbc-8580-14394273fcc6)

**Comment by Yyang-sys** [comment]
哈喽各位，Sen2_MTC_New数据集的存放路径和文件结构是什么
